src/reconcile/rec_write.c

```diff
@@ -1268,9 +1276,11 @@ __rec_txn_read(WT_SESSION_IMPL *session, WT_RECONCILE *r,
 		if (*updp == NULL && r->las_skew_newest)
 			*updp = upd;
 
-		if (F_ISSET(r, WT_REC_VISIBLE_ALL) ?
+		/* Consider non durable updates as unstable. */
```

Comment:
Could you replace `unstable` with `uncommitted` for consistency? i.e: switch to using the same comment as on line `1245`